website/cue/reference/components/sinks/influxdb_logs.cue

```diff
@@ -64,6 +64,19 @@ components: sinks: influxdb_logs: {
 				examples: ["service"]
 			}
 		}
+		tags: {
+			required:    false
+			common:      false
```

Comment:
And a question, what does this `common` do?

Reply:
`common` is used in our docs for options/configuration that would be "commonly used" by the operator - or left default/unused otherwise.